helm-chart/dicom-adapter/files/appsettings.json

```diff
@@ -19,10 +19,28 @@
       "temporary" : "/payloads",
       "watermarkPercent": 85,
       "reserveSpaceGB": 5
+    },
+    "services": {
+      "platform": {
+        "uploadMetadata": false,
+        "metadata": [
+          "0008,0020",
+          "0008,0060",
+          "0008,1030",
+          "0008,103E",
+          "0010,0020",
+          "0010,0030",
+          "0010,1010"
+        ]
+      }
     }
   },
   "Logging": {
     "LogLevel": {
+      "Nvidia": "Information",
+      "Nvidia.Clara.DicomAdapter.Server.Services.Disk.StorageInfoProvider": "Information",
```

Comment:
Is there a documented comprehensive list of `Services` that can be fine-tuned here?

Reply:
No, but I have updated the default `appsettings.json` to include major services/namespaces.